README.md

```diff
@@ -76,6 +76,17 @@ Flags:
                              Period of time to store data for
 ```
 
+### Environment variables
+
+```
+ROTATION_* variables are used to configure log rotation with logrotate
+
+      ROTATION_FILE_LOG: "/var/log/cleafy/metrics/metrics.prom" # must be the some of output parameter in PROM_ARGS envVar
+      ROTATION_PERIOD: daily
+      ROTATION_COUNT: 10 # after 10 periods will delete older log files
+      ROTATION_SIZE: -1
+      PROM_ARGS: "--output=/var/log/cleafy/metrics/metrics.prom --interval=30s -u serviceName1=URL1 -u serviceName2=URL2"
```

Comment:
if the above are indented for the rotation, this is not related to rotation. I would propose something like this:


- `PROM_ARGS`: The argument for the promqueen service. Output, interval and at least one service is mandatory. E.g. `--output=/var/log/cleafy/metrics/metrics.prom --interval=30s -u serviceName1=URL1 -u serviceName2=URL2 ...`
- `ROTATION_FILE_LOG`:  where the rotation should occurr. Must be the same of the "output" parameter in PROM_ARGS
- `ROTATION_PERIOD`: how frequently a rotation will occurr. E.g. "daily"
- `ROTATION_COUNT`: how many rotation will be retained. E.g. 10
- `ROTATION_SIZE`: how big each rotation file will be in bytes. -1 means no limit
```

Maybe we should provide also an example `docker run` command with the least amount of variables needed to start it, what do you think?

Reply:
thank you for feedback, looks better and clearer!